core/src/main/scala/scala/pickling/PicklingErrors.scala

```diff
@@ -0,0 +1,165 @@
+package scala.pickling
+
+private[pickling] object Feedback {
+
+  def failedGenerationMsg(x: String, y: String, z: String) =
```

Comment:
can we use different arg names than x/y/z?   I'll be confused everytime I look at this method :)